src/app.js

```diff
@@ -1 +1,42 @@
 'use strict';
+/* eslint-disable no-console */
+
+const { terminal } = require('./terminal');
+const { generateNumber } = require('./generateNumber');
+const { countBullsAndCows } = require('./countBullsAndCows');
+const {
+  WIN_MESSAGE,
+  INVALID_INPUT_MESSAGE,
+  NUMBER_MESSAGE,
+  EXPECTED_NUMBER_LENGTH,
+} = require('./constants');
+
+const randomNumber = generateNumber();
+
+function playBullsAndCows() {
+  terminal.question(NUMBER_MESSAGE, (userInput) => {
+    const userNumber = userInput;
+
+    if (userNumber === randomNumber) {
+      console.log(WIN_MESSAGE);
+      terminal.close();
+
+      return;
+    }
+
+    if (typeof +userInput !== 'number'
+      || [...new Set(userInput.split(''))].length !== EXPECTED_NUMBER_LENGTH) {
```

Comment:
Create function for this, to check if number is valid
Because to write all this code in `if` is a bad idea